src/main/java/org/apache/commons/lang3/CharSequenceUtils.java

```diff
@@ -195,6 +196,7 @@ static int lastIndexOf(final CharSequence cs, final int searchChar, int start) {
                     return i;
                 }
             }
+            return NOT_FOUND;
```

Comment:
mm . Maybe it's my fault. 
The idea it's exit the loop and return -1 (NOT_FOUND) if not found the charset

Reply:
ah, I see, this is a new line, and the other mods are updates. ok, I'll take a look tomorrow probably...

Reply:
sorry. my fault for mixing.